qbft/timeout.go

```diff
@@ -13,6 +13,14 @@ var (
 	CutoffRound = 15 // stop processing instances after 8*2+120*6 = 14.2 min (~ 2 epochs)
 )
 
+const (
+	// AttestationCutOffRound is the round after which the instance will stop processing attestations
+	AttestationCutOffRound = 12 // stop processing attestations after 8*2+120*3 = 6.2 min (~ 1 epoch)
+
+	// SyncCommitteeCutOffRound is the round after which the instance will stop processing sync committee messages
+	SyncCommitteeCutOffRound = 4 // stop processing sync committee messages after one slot
+)
```

Comment:
> this can be removed since its replaced with Config.GetCutOffRound

It can't be removed...
These are the consensus parameters...
It makes no sense to have nodes configure different values...
Unfortunately the correct specifications and tests for timer were never merged due to lacking priority so it looks weird... Maybe we can try to get it up in priority soon!

Reply:
> btw these aren't needed now because the spec no longer stops duties, right?

hold on.. Before we stopped duties because the spec specified that the same runner object should be used.
But still no point to run after those cutoffs...

Just to be clear, you can still in impl reuse the same objects (to save resources) as long as all protocol transitions will be compliant with spec